LinkedInSignIn/ApiDefinition.cs

```diff
@@ -25,32 +25,28 @@ namespace LinkedIn.SignIn
 	delegate void DeeplinkErrorBlock(NSError arg0, string arg1);
 
 
-
-	// typedef void (^APIErrorBlock)(LISDKAPIResponse *, NSError *);
-
 	// @interface LISDKSessionManager : NSObject
-	[BaseType(typeof(NSObject))]
-	interface LISDKSessionManager
+	[BaseType(typeof(NSObject), Name = "LISDKSessionManager")]
+	interface SessionManager
 	{
 		// +(instancetype)sharedInstance;
 		[Static]
 		[Export("sharedInstance")]
-		LISDKSessionManager SharedInstance();
+		SessionManager SharedInstance { get; }
```

Comment:
You can change these types of methods (`Type MethodName()`) to getter properties